src/screens/ExpensesScreen.js

```diff
@@ -3,17 +3,19 @@ import React, { useState, useEffect } from 'react';
 import { db, auth } from "../firebase";
 import { query, collection, doc, onSnapshot, deleteDoc } from "firebase/firestore";
 import { ExpenseEntry } from '../Components/ExpenseEntry';
+import { MonthPicker } from "../Components/MonthPicker";
```

Comment:
perhaps these can be combined into 

`import { MonthPicker, ExpenseEntry } from "../Components";`

Reply:
I've made the changes!